test/async-await.js

```diff
@@ -65,6 +65,43 @@ function asyncAwait (Queue, test) {
       t.is(order.shift(), 7)
     })
   })
+
+  test('Drain hook', t => {
+    t.plan(7)
+
+    const q = Queue()
+    const order = [1, 2, 3, 4, 5]
+
+    q.drain(async () => {
+      await sleep(100)
+      t.ok('called')
+    })
+
+    q.add(async q => {
+      t.is(order.shift(), 1)
+    })
+
+    q.add(async q => {
+      t.is(order.shift(), 2)
+
+      q.add(async q => {
+        t.is(order.shift(), 3)
+      })
+
+      q.drain(async () => {
+        await sleep(100)
```

Comment:
why is it necessary?

Reply:
Otherwise I cannot test the async function, check the older coverage test.

Reply:
In the old test https://github.com/delvedor/workq/commit/754ea7ae32aa5d4bd0441c90681b5e5ab36b62bb#diff-e0c0566de1e15591af4e142dd5ac670b it makes sense to me because you are waiting that the previous task was finished but here you define a hook it should just be called after the queue is empty?